README.md

```diff
@@ -1,2 +1,31 @@
 # vc-api-issuer-test-suite
 Test Suite for Issuers that implement the VC HTTP API
+
+## Table of Contents
+
+- [Background](#background)
+- [Install](#install)
+- [Usage](#usage)
+- [Implementation](#implementation)
+
+
+## Background
+
+Provides interoperability tests for issuers that support [VC-API](https://w3c-ccg.github.io/vc-api/).
+
+## Install
+
+```js
+npm i
+```
+
+## Usage
+
+```
+npm test
```

Comment:
this should also include some info regarding the client secrets that are required for running the tests for certain implementations.

Reply:
addressed here: https://github.com/w3c-ccg/vc-api-issuer-test-suite/pull/5/commits/bfcddfffaa8a0a98f4b8cee490344aa54217b4c6